packages/walletconnect/src/index.ts

```diff
@@ -0,0 +1,274 @@
+import {
```

Comment:
The walletconnect provider has been updated after this PR was published, so you need to apply those changes in this file to avoid a regression

Reply:
updated but please review that I didn't screw up the rebase